compiler/src/main/java/motif/compiler/errors/parsing/NullableSpreadMethodHandler.kt

```diff
@@ -0,0 +1,18 @@
+package motif.compiler.errors.parsing
+
+import motif.ast.compiler.CompilerMethod
+import motif.compiler.errors.ErrorHandler
+import motif.models.errors.NullableSpreadMethodError
+import javax.lang.model.element.Element
+
+class NullableSpreadMethodHandler : ErrorHandler<NullableSpreadMethodError>() {
+
+    override fun message(error: NullableSpreadMethodError): String {
+        return "@Spread is not allowed on classes with @Nullable methods. ${error.spreadable.qualifiedName} has @Nullable methods."
```

Comment:
Would be good to add a recommendation: "Return an `Optional<>` type instead"

Reply:
Good point, done